pkgs/top-level/all-packages.nix

```diff
@@ -21462,6 +21457,10 @@ with pkgs;
                       mount wall hostname more sysctl getconf
                       getent;
 
+  fts = if hostPlatform.isMusl then netbsd.fts else null;
```

Comment:
Maybe there is some `meta.available` thing you can do instead of `null`?

Reply:
It should be provided by most libcs except for Musl. It will build everywhere, just runs on Musl.